src/include/platform/CHIPDeviceConfig.h

```diff
@@ -617,18 +617,56 @@
 #define CHIP_DEVICE_CONFIG_BLE_ADVERTISING_INTERVAL_CHANGE_TIME 30000
 #endif
 
+/**
+ * CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING
+ *
+ * Optional configuration to enable Extended Announcement Duration up to 48h
+ * Should be used together with extending CHIP_DEVICE_CONFIG_DISCOVERY_TIMEOUT_SECS past 15 minutes
+ * Disabled by default
+ */
+
 #ifndef CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING
 #define CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING 0
 #endif
 
 #if CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING
 
+/**
+ * CHIP_DEVICE_CONFIG_BLE_EXT_ADVERTISING_INTERVAL_CHANGE_TIME
+ *
+ * The amount of time in miliseconds after which BLE advertisement should be switched from the slow
+ * advertising to the extended advertising, counting from the moment of advertisement commencement.
+ *
+ *  Defaults to 15 minutes measured in ms.
```

Comment:
Please measure in seconds everywhere or milliseconds everywhere.

Reply:
Its all measured in ms, the minutes where just documentation. I will adjust it to the ms value for consistency.